accepted/2020/net5/net5.md

```diff
@@ -1020,13 +1027,24 @@ In MSBuild you can't easily do comparisons like:
 <ItemGroup Condition="'$(TargetFramework)' >= 'net5.0'`">
 ```
 
-because that would be a string comparison. Rather, you need to do comparisons
+because that would be a string comparison. Rather, you need to do a comparison
 like this:
 
 ```xml
-<ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(TargetFrameworkVersion)' >= '3.0'">
+<ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(_TargetFrameworkVersionWithoutV)' >= '3.0'">
```

Comment:
The proper way to do this is using `VersionLessThan` msbuild function, like: `$([MSBuild]::VersionLessThan($(TargetFrameworkVersion), '3.0'))`